server/webapp/WEB-INF/rails.new/webpack/views/dashboard/pipeline_operations_widget.js.msx

```diff
@@ -89,19 +89,33 @@ const PipelineOperationsWidget = {
       modal.render();
     };
 
+    const getErrorMessageFromStatus = (textStatus) => {
+      if (textStatus === 'parsererror') {
+        return "Error occurred while parsing the data.";
```

Comment:
how about `Error occurred while fetching trigger with options information for pipeline "up42". Check server logs for more information.`